stdlib/Profile/src/Profile.jl

```diff
@@ -256,6 +256,10 @@ function callers(funcname::String, bt::Vector, lidict::LineInfoFlatDict; filenam
     end
 end
 
+callers(funcname::String, bt::Vector, lidict::LineInfoDict; kwargs...) =
+    callers(funcname, flatten(bt, lidict)...; kwargs...)
+
+
```

Comment:
Weird number of blank lines—either zero (since the following ones are together) or one.